src/algebraic_geometry/locally_ringed_space.lean

```diff
@@ -117,6 +115,20 @@ instance : category LocallyRingedSpace :=
   id_comp' := by { intros, ext1, simp, },
   assoc' := by { intros, ext1, simp, }, }.
 
+lemma comp_val {X Y Z : LocallyRingedSpace} (f : X ⟶ Y) (g : Y ⟶ Z) :
+  (f ≫ g).val = f.val ≫ g.val := rfl
+
+lemma comp_val_c {X Y Z : LocallyRingedSpace} (f : X ⟶ Y) (g : Y ⟶ Z) :
+  (f ≫ g).val.c = g.val.c ≫ (presheaf.pushforward g.val.base).map f.val.c := rfl
+
+lemma comp_val_c_app {X Y Z : LocallyRingedSpace} (f : X ⟶ Y) (g : Y ⟶ Z) (U) :
+  (f ≫ g).val.c.app (op U) = g.val.c.app (op U) ≫ f.val.c.app (op $ (opens.map g.val.base).obj U)
+:= rfl
+
+lemma comp_val_c_app' {X Y Z : LocallyRingedSpace} (f : X ⟶ Y) (g : Y ⟶ Z) (U) :
+  (f ≫ g).val.c.app U = g.val.c.app U ≫ f.val.c.app (op $ (opens.map g.val.base).obj U.unop)
+:= rfl
```

Comment:
Should these be simp lemmas (with reassoc and elementwise)?

Reply:
I once used the last lemma but no longer need them. Should I delete them?

Reply:
I think keeping the fourth and removing the third is sufficient.

Reply:
Actually I still use the last lemma. I removed the third and labelled the others with `@[simp]`. However I didn't add `reassoc` and `elementwise` as analogous lemmas for presheafed spaces and [sheafed spaces](https://github.com/leanprover-community/mathlib/blob/d9cea393084fe7896ced66322883576043cdf3aa/src/algebraic_geometry/sheafed_space.lean#L90-L92) don't have them.